docs/policy-only.md

```diff
@@ -0,0 +1,170 @@
+# Running Antrea In Policy Only Mode
+
+Antrea supports chaining with routed CNI implementations such as EKS CNI. In this mode, Antrea
```

Comment:
It is still not clear to me, what routed CNI means. It does not seem a well known term. Also does policy only mode works with only routed CNI?
Are line 18-25 define what is a routed CNI? If so, might be better make it more explicit. @suwang48404

Reply:
we had switch CNI before (azure). but today antrea supports only aws.

thx

Reply:
I believe @reachjainrahul made NetworkPolicy only mode for AKS Engine as well (it is already merged to AKS Engine). Is that routed too?
Again might be better to make it clear what is a routed CNI.

Reply:
1. AKS engine is not AKS. 
2. snippet of document, does that not explaining what *is* routed CNI?

The diagram on the left illustrates a routed CNI network topology such as AWS EKS. In this topology a Pod connects to the host network via a point-to-point(PtP) like device, such as (but not limited to) a veth-pair. On the host network, a host route with corresponding Pod's IP address as destination is created on each PtP device. Within each Pod, routes are configured to ensure all outgoing traffic is sent over this PtP device, and incoming traffic is received on this PtP device. This is a spoke-and-hub model, where to/from Pod traffic, even within the same worker Node must traverse first to the host network and be routed by it.

Reply:
Do you mean the whole paragraph are must to identify a routed CNI, and the current NetworkPolicy only mode requires all these CNI characteristics to work?

Reply:
1. yes, the whole paragraph explains what routed CNI topology is. 
2. yes, Antrea network PolicyOnly mode *must* has this type of topology (i.e aws, or aks engine with routed CNI configured)

Reply:
Ok. Let us see if @antoninbas has further comments or not.